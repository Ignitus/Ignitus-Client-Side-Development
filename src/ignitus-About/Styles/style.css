._align{
	text-align: center;
}

._started {
	font-size: 23px;
    font-weight: 400;
    margin-top: 70px;
    margin-bottom: 40px;
}

._aboutus{
	font-size: 16px;
    font-weight: 400;
    margin:40px auto;
}

._about{
	font-weight: 800 !important;
    font-size: 36px !important;
    color: #0d0465 !important;
    margin-top: 115px;
}

