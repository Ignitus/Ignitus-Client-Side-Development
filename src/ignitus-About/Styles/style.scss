// import ignitus token variables
@import './src/shared/styles/ignitus-tokens.scss';
.about-wrapper {
	font-size: $ignite-font-size-sm;
	font-weight: $ignite-font-weight-normal;
	background: url('https://storage.googleapis.com/ignitus/Assets/split-blue-background.png') no-repeat;
	background-size: 70%;
	background-position: top right;
}

.about {
	box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.25);
	border-radius: 16px;
}
