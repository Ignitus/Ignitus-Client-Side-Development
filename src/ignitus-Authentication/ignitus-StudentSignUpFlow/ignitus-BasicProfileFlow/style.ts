import styled from '@emotion/styled';
import { Icon as I } from '../../../ignitus-Shared/ignitus-Utilities/Components/icon';
import { Paragraph } from '../../../ignitus-Shared/ignitus-DesignSystem/shared';
import { Input } from '../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultInput/styles';
import { DefaultInput, IgnitusBlue, XL } from '../../../ignitus-Shared';
import { DefaultButtonWithIcon } from '../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultButtonWithIcon/Components/index';

export const Icon = styled(I)`
  width: 1.5rem;
  fill: ${IgnitusBlue};
`;

export const BottomSection = styled.div`
  display: flex;
  padding: 4rem;
  width: 100%;
`;

export const InputContainer = styled.div`
  display: flex;
  flex-direction: column;
  flex: 2;
`;

export const SubTitle = styled(Paragraph)`
  margin-top: 1rem;
  font-size: ${XL};
`;

export const ImageContainer = styled.div`
  img {
    height: 10rem;
    width: 10rem;
    border-radius: 0.9rem;
  }
`;

export const LeftSection = styled.div`
  display: flex;
  flex-direction: row;
  flex: 1;
  padding: 2rem;
`;

export const RightSection = styled.div`
  display: flex;
  flex-direction: column;
  margin-left: 1rem;
  padding: 2rem;
`;

export const Row = styled.div`
  display: flex;
`;

export const Text = styled(Input)`
  margin: 0.2rem;
`;

export const StyledDefaultInput = styled(DefaultInput)`
  padding: 0.5rem;
  padding-left: 1rem;
  margin: 0 0.5rem 0.5rem;
`;

export const StyledDefaultButtonWithIcon = styled(DefaultButtonWithIcon)`
  width: 12rem;
`;
