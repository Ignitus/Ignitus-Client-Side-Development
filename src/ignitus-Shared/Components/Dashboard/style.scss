@import '../../styles/ignitus-components-common.scss';
@import '../../styles/ignitus-tokens.scss';

.navigator {
  position: absolute;
  width: 220px;
  height: 320px;
  left: 75px;
  top: 145px;
  background: $ignite-color-white;
  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1);
  border-radius: 16px;
  padding: 20px 0;
}
.option {
  font-size: 16px;
  font-weight: bold;
  color: $ignite-color-blue-bg;
  padding: 10px 30px;
}
svg {
  margin-right: 25px;
}

svg g rect {
  fill: $ignite-color-blue-bg;
}

.option:hover {
  background-color: $ignite-color-blue-bg;
  color: $ignite-color-white;
}
.option:hover svg g rect {
  fill: $ignite-color-white;
}
