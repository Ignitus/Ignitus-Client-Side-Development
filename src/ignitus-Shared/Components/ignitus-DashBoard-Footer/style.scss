@import '../../styles/ignitus-components-common.scss';
@import '../../styles/ignitus-tokens.scss';

.hodo {
  position: absolute;
  height: 121px;
  width: 276px;
  left: 1072px;
  top: 116px;
  background: $ignite-background-color-white;
  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1);
  border-radius: 16px;
}

.column {
  line-height: 16px;
  display:flex;
  flex-direction:column;
  align-items:center;
  justify-content: space-between;
}

a {
  font-family: $ignite-font-family-default;
  font-weight: $ignite-font-weight-normal;
  font-style: normal;
  font-size: 14px;
  color: $ignite-color-gray-20;
}

a:hover {
  color: $ignite-color-blue-bg;
}

.columnOne {
  padding:10px;
}

.columnTwo {
  padding: 8px;
}

.columnThree {
  font-size: 14px;
  color: $ignite-color-blue-bg;
  padding: 8px;
}
