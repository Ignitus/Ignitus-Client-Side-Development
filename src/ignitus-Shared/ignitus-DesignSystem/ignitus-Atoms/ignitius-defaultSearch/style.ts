import styled from '@emotion/styled';
import * as C from '../colors';
import * as F from '../fonts';


export const Input = styled.input`
  height: 40px;
  width: 300px;
  background-color: ${C.GreyBackground};
  padding-left: 3px;
  color: #000066;
  font-weight: ${F.SemiBold};
  font-size: ${F.XL};
  border: none;
  border-radius: 10px;
  ::placeholder {
    color: ${C.IgnitusBlue};
    font-weight: ${F.SemiBold};
    font-size: ${F.XL};
  }
`;