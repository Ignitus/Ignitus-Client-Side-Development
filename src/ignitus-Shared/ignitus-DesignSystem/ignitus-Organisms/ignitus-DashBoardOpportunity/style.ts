import styled from '@emotion/styled';
import * as T from '../../ignitus-Atoms/typography';
import * as C from '../../ignitus-Atoms/colors';
import { flexibleRowDiv } from '../../shared';
import { Normal, XS } from '../../ignitus-Atoms/fonts';

export const Container = styled.div`
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  background-color: ${C.White};
  border-radius: 20px;
  display: flex;
`;

export const Row = styled(flexibleRowDiv)``;
export const LeftRow = styled.div`
  flex: 1;
  img {
    max-width: 100%;
    max-height: 100%;
    border-top-left-radius: 20px;
    border-bottom-left-radius: 20px;
  }
`;

export const RightRow = styled.div`
  display: flex;
  flex-direction: column;
  flex: 3;
  margin-left: 20px;
  padding: 0.5rem;
`;

export const Description = styled(T.Paragraph)`
  font-weight: ${Normal};
  color: ${C.GreyText};
  margin-right: 1.5rem;
  margin-top: 0.5rem;
`;

export const Title = styled(T.Heading3)`
  flex: 1;
`;

export const SubTitle = styled.p`
  font-weight: ${Normal};
  color: ${C.GreySecondaryText};
  font-size: ${XS};
`;
