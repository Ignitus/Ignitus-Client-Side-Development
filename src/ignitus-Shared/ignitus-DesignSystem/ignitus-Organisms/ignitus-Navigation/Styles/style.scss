$base-color: #000e8c;
$hover-color: #f9bf64;

.navbar {
  align-items: center;
  display:flex;
  position: fixed !important;
  width: 100%;
  top: 0;
  left: 0;
  z-index: 9001;
  justify-content: space-between;
  transition: background-color 300ms ease-in;
}

.navlinks{
  display: flex;
  justify-content: center;
  align-items: center;
  margin-right:1rem;
}

.navlinks-item{
  padding:0rem 1rem;
  align-self: stretch;
}

.navlink{
  color: #fff;
  font-size: 14px;
  font-weight: 700;

  &:hover {
    color: $hover-color;
  }
}

#burger{
  width:1.5rem;
  height:1.5rem;
  position: absolute;
  top:1.2rem;
  right:2rem;
  display:none;
  cursor:pointer;
}

.line{
  display:block;
  background-color: $base-color;
  padding:0.1rem 0.8rem;
  width:0.1rem;
  margin:0.3rem;
  border-radius:1rem;
  transition:transform 100ms ease-in-out;
}

.navbar.whitenav{
  background-color:#fff;
  .navlink{
    color: $base-color;

    &:hover {
      color: $hover-color;
    }
  }
}

.line.mobile:first-child{
  transform: rotateZ(46deg) translateY(0.3rem) translateX(1px);
}

.line.mobile:last-child{
  transform: rotateZ(-43deg) translateY(-0.3rem);
}

.line.mobile:nth-child(2){
  display:none;
}

@media only screen and (max-width:1042px){

  #burger{
    display:block;
    top:1.2rem;
  }

  .navbar{
    flex-direction: column;
    background:#fff;
  }

  .navlinks.mobile{
    display:none;
  }

  .navlinks{
    flex-direction: column;
    align-items:center;
    justify-content: center;
    width:100%;
  }

  .navlinks-item{
    margin-left:0rem;
    margin-bottom:1rem;
  }

  .navbar-brand{
    align-self: flex-start;
    margin-left:5px;
  }

  .navlink{
    color: $base-color;
  }

}
