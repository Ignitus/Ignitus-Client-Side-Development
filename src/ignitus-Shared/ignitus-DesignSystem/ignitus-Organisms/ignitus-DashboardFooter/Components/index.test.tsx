import React from 'react';
import { mount, render } from 'enzyme';
import emotionSerializer, { matchers } from 'jest-emotion';
import { DashboardFooter } from './index';
import { Copyright } from '../style';

expect.addSnapshotSerializer(emotionSerializer);
expect.extend(matchers);

it('should rendered', () => {
  const tree = render(<DashboardFooter />);
  expect(tree).toMatchSnapshot();
});

describe('<DashboardFooter />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(<DashboardFooter />);
  });

  it('should render ul, li, p tags', () => {
    expect(wrapper.find('ul').length).toBe(2);
    expect(wrapper.find('li').length).toBe(5);
    expect(wrapper.find('p').length).toBe(1);
  });

  it('should render 2018 copyright text', () => {
    expect(wrapper.find(Copyright).text()).toEqual('Â© Ignitus 2018');
  });
});
