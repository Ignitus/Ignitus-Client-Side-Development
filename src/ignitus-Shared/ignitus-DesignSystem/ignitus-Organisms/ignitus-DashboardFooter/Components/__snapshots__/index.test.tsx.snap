// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rendered 1`] = `
.emotion-6 {
  background-color: #ffffff;
  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.05);
  border-radius: 16px;
  padding: 1rem;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-0 {
  color: #4D4D4D;
  font-size: 14px;
  font-family: Raleway,sans-serif;
  padding: 0.4rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-0 li {
  padding: 0rem 1rem;
}

.emotion-0 li a:link,
.emotion-0 li a:visited {
  color: #3d3d3d;
}

.emotion-0 li a:hover,
.emotion-0 li a:active {
  color: #000066;
}

.emotion-4 {
  color: #000066;
  font-family: Raleway,sans-serif;
  font-size: 14px;
  text-align: center;
  margin-top: 15px;
  margin-bottom: 0;
}

<footer
  class="emotion-6"
>
  <div
    class="emotion-5"
  >
    <div
      class="emotion-1"
    >
      <ul
        class="emotion-0"
      >
        <li>
           
          <a
            href="#!"
          >
            About
          </a>
           
        </li>
        <li>
           
          <a
            href="#!"
          >
            Privacy
          </a>
        </li>
        <li>
           
          <a
            href="#!"
          >
            Careers
          </a>
           
        </li>
      </ul>
    </div>
    <div
      class="emotion-1"
    >
      <ul
        class="emotion-0"
      >
        <li>
           
          <a
            href="#!"
          >
            Help Center
          </a>
           
        </li>
        <li>
           
          <a
            href="#!"
          >
            Terms & Condtions
          </a>
        </li>
      </ul>
    </div>
    <p
      class="emotion-4"
    >
      Â© Ignitus 2018
    </p>
  </div>
</footer>
`;
